packages/components/src/components/buttons/Button/Button.tsx

```diff
@@ -48,8 +48,7 @@ export const ButtonContainer = styled.button<ButtonContainerProps>`
     &:disabled {
         background: ${({ theme }) => theme.BG_GREY};
         color: ${({ theme }) => theme.textDisabled};
-        pointer-events: none;
-        cursor: default;
+        cursor: not-allowed;
```

Comment:
I suggest completely not use this mechanism and rely solely on `onClick={ isDisabled ? undefined : onClick }`

Is there anything this is useful? @hstastna / @jvaclavik (Genuine question, I am oldchool grumpy dev who hates CSS :D)

Reply:
I am already about to use `onClick={ isDisabled ? undefined : onClick }` as it's definitely better as we make sure `onClick` isn't called when component disabled.
And what `cursor: not-allowed;` does is that we just can see a different, nicer cursor for disabled component when hovering on it.